### DISCUSSION TITLE
How Memoria pulls out data from the game

### DISCUSSION THREAD

**Discussion by FlameHorizon** [Albeoris/Memoria#198] [Q&A] [answered]
I would like ask you, where I have to look within this repository to know, how c# is getting values from the game memory and putting it back? Right now, I'm away from my _battle station_ and I'm having hard time finding it in source code. Thanks.

**Comment by Tirlititi** [comment]
I don't understand what is the "game memory" that you are talking about (and what it has to do with your "battle station").

**Reply by FlameHorizon** [reply]
I would like to know how or where, in the code, it reads/writes data to the game.

**Reply by Albeoris** [reply]
Doesn't do it at all. This is not a mod, but a recreated game engine. This is what Unity uses as the entry point. All logic is here, all game data is here. There is no interaction with any other applications. Everything you need to implement your ideas is already in the repository.

**Reply by Albeoris** [reply]
However, the engine has a PS-byte code interpreter. EventEnginge - it executes all battle scripts and events on the map. It has its own "memory" and execution stack. I have a long-running C# [bytecode rewrite project ](https://pastebin.com/qvnYqmEL) inspired by Tirlititi's pseudo-language, but I'm not sure I'll ever finish it.

**Reply by FlameHorizon** [reply]
So Unity is just using FF9 game assets, and that is it. All logic related thing are already in a code on this repository. Am I right?
